### ISSUE TITLE
Missing yarn lock file for example-install-command on v10

### ISSUE THREAD

**Issue by MikeMcC399** [cypress-io/github-action#697]
## Problem description

When the workflow [example-install-command.yml](https://github.com/cypress-io/github-action/blob/master/.github/workflows/example-install-command.yml) is run, the `install-command-v10` job shows a warning concerning the lock file found:

```text
install command command "yarn --frozen-lockfile --silent"
current working directory "/home/runner/work/github-action/github-action/examples/v10/install-command"
/usr/local/bin/yarn --frozen-lockfile --silent
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
```

## Steps to reproduce

Push to `master` to trigger [example-install-command.yml](https://github.com/cypress-io/github-action/blob/master/.github/workflows/example-install-command.yml)

Search for the logs with https://github.com/cypress-io/github-action/actions/workflows/example-install-command.yml?query=branch%3Amaster and then view the job "install-command-v10" > "Custom Yarn command" noticing the warning from `yarn`.

## Suggested fix

- Delete the file [examples/v10/install-command/package-lock.json](https://github.com/cypress-io/github-action/blob/master/examples/v10/install-command/package-lock.json)
- Execute `yarn install cypress@10.0.0 -D` in directory [examples/v10/install-command/](https://github.com/cypress-io/github-action/blob/master/examples/v10/install-command/) to generate `yarn.lock` file and keep Cypress at the same version level (10.0.0) as the other v10 examples.

**Comment by MikeMcC399** [comment]
- PR #698 submitted to resolve this issue

**Comment by MikeMcC399** [comment]
This issue is now resolved and the log [6610708399](https://github.com/cypress-io/github-action/actions/runs/3876635618/jobs/6610708399) no longer shows the warning:

```text
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
```
